src/basic/build.h

```diff
@@ -133,6 +135,8 @@
 #define _IDN_FEATURE_ "-IDN"
 #endif
 
+#define _CGROUP_HIEARCHY_ "default=" STRINGIFY(DEFAULT_HIERARCHY)
```

Comment:
Hm, I'm not convinced. Our other output is different, because we show what is compiled in and what is compiled out. (E.g. +XZ +LZ4 means that lz4 will be used). So this would be long and misleading. I now (before I read your comment), reworked the patch to say "default-hierarchy=legacy|hybrid|unified". I think that's clear and concise, please have a look.